App/Containers/PhotoViewerScreen.js

```diff
@@ -116,7 +119,7 @@ const mapStateToProps = (state, ownProps) => {
 
 const mapDispatchToProps = (dispatch) => {
   return {
-    share: (hash) => { dispatch(IpfsActions.shareImageRequest('beta', hash)) },
+    share: (item) => { dispatch(IpfsActions.shareImageRequest('beta', item.hash)) },
```

Comment:
This this is broken because above in `sharePressed` you're passing in `item.hash` and here you're trying to use `item.hash`.

Reply:
doh. where do you see it passing in a item.hash?